platform/device-manager.ts

```diff
@@ -3,24 +3,18 @@
  * TODO(cjdaly) integrate with Mock Local Home Platform and Mock Network
  **/
 /// <reference types="@google/local-home-sdk" />
+import { MockNetwork } from './mock-radio';
 
-import { DeviceManager } from "@google/local-home-sdk";
-
-export class DeviceManagerStub implements DeviceManager{
-
-  markPending(request: smarthome.IntentRequest): Promise<void> {
-    //TODO(cjdaly) implementation
-    throw new Error("Method not implemented.");
-  }
-
-  getProxyInfo(id: string): smarthome.ProxyInfo {
-    //TODO(cjdaly) implementation
-    throw new Error("Method not implemented.");
-  }
-
+export class DeviceManagerStub implements smarthome.DeviceManager {
   deviceId: string;
   error: boolean;
   commands: smarthome.DataFlow.TcpRequestData[] = [];
+  private mockNetwork: MockNetwork;
+
+  constructor(mockNetwork: MockNetwork) {
+    this.mockNetwork = mockNetwork;
```

Comment:
how is this being used?

Reply:
This would be part of #23 to allow the fulfillment app to utilize the `MockNetwork`.  I'll remove from this PR.

Reply:
Done